docs/design/y-flow/y-flow-nb-api.md

```diff
@@ -317,6 +317,57 @@ Response body:
 }
 ```
 
+## Ping operation
+
+REST endpoint: `PUT /v2/y-flows/{y_flow_id}/ping`
```

Comment:
I'd use POST method to follow to the command pattern (https://github.com/telstra/open-kilda/blob/develop/docs/guidelines/api-guidelines.md#command-pattern).

Reply:
@sergii-iakovenko changed to POST